src/tests/ftest/control/dmg_pool_query_ranks.py

```diff
@@ -53,6 +53,7 @@ def test_pool_query_ranks_basic(self):
         self.log_step("Checking pool query with dead ranks state information")
         data = self.dmg.pool_query(self.pool.identifier, health_only=True)
         self._verify_ranks([], data, "dead_ranks")
+        self._verify_ranks([], data, "disabled_ranks")
```

Comment:
Shouldn't we be making this change to master before implementing it on the release/2.6 branch?  Also lines 58-60 of this file don't exist on https://github.com/daos-stack/daos/blob/master/src/tests/ftest/control/dmg_pool_query_ranks.py

Reply:
@phender: I'm confused. Are these tests broken on master, too? Or just release/2.6? There are significant differences between the branches (2.6 does not have @knard38's changes to allow for enabled and disabled ranks to be queried simultaneously, in particular).

Reply:
Sorry, I looked at my original back-port PRs, I missed "Feature: pool" tags in the following commits which missed this test. it was included in the first rename commit.

Reply:
Yes, the test is only failing in the release/2.6 branch, but it is not clear to me with which commits the tests deviated and where the cherry-picks went wrong to cause this.  I just want to be sure we're not making this more confusing by just patching the test failure and possibly removing test functionality or coverage.

Reply:
For this particular change, I suspect that @wangshilong just made a small mistake in resolving a conflict. The health-only pool query is supposed to return a `disabled_ranks` array that can be empty on this branch. As the query also always returns a `dead_ranks` array on this branch, the test should check both.

In any case, I think that this is all working as intended on this branch.